IM_calculation/scripts/split_IESD.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import argparse
+from pathlib import Path
+
+g=9.81
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        "Script for retrieving IESD outputs from the single IM csv and storing them in separate files depending on model"
```

Comment:
This script should refer to SDI instead of IESD, unless ratios are being calculated (I don't think they are)

Reply:
Also, will this script be required once SDI moves to adv_im?

Reply:
This script is no longer needed. Will remove it.  
(haha. my second monitor is broken, and lots of stuff escaped my check )